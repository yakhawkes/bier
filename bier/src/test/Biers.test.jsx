import React from 'react'
import renderer from 'react-test-renderer';

const Biers = require('../app/components/Biers.jsx')
const api = require('./apiclient')

test('Page of Biers', () => {
  const component = renderer.create(
    <Biers api={api} />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
