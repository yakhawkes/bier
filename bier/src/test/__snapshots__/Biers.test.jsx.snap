// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page of Biers 1`] = `
<div>
  <header
    className=""
  >
    <h1
      id="logo"
    >
      Bier!
    </h1>
    <nav>
      <form
        onSubmit={[Function]}
      >
        <input
          autoComplete="off"
          id="querytext"
          onChange={[Function]}
          placeholder=""
          type="text"
          value=""
        />
        <button
          disabled={true}
        >
          Submit
        </button>
      </form>
      <a
        href=""
        onClick={[Function]}
      >
        name
        
        
      </a>
      <a
        href=""
        onClick={[Function]}
      >
        abv
        
        
      </a>
      <a
        href=""
        onClick={[Function]}
      >
        status
        
        
      </a>
    </nav>
  </header>
  <div>
    <div
      className="bierlist"
    />
  </div>
  <div
    className="morebeer"
  >
    No More bier! :(
  </div>
</div>
`;
